build/constants.yaml

```diff
@@ -430,7 +430,7 @@
 - name: staging-runtime-constants
   constants:
     dotnetcore-staging-runtime-versions:
-    -
+    - 
```

Comment:
Could you please provide more context? Was the version without the space not a valid YAML?

Reply:
It's a valid YAML. But since Automation checks changes to `constants.yaml` for detecting if new versions are available. Having a space discrepancy in `constants.yaml` leads Automation to think there are new versions.

Reply:
YamlDotNet library, in Automation, expects there to be a space after the `-`

Reply:
Thanks for clarifying!